lib/shroud/plugin.rb

```diff
@@ -115,14 +139,19 @@ def reportJacoco(
     #          if true, will fail builds that are under the provided thresholds per file. If false, will only warn.
     #          default true.
     #
+    # @param   [Symbol] coverageType
+    #          defines the type of coverage to use.
+    #          default instruction.
+    #
     # @return  [void]
     def reportKover(
       moduleName:,
       file:,
       totalProjectThreshold: 90,
       modifiedFileThreshold: 90,
       failIfUnderProjectThreshold: true,
-      failIfUnderFileThreshold: failIfUnderProjectThreshold
+      failIfUnderFileThreshold: failIfUnderProjectThreshold,
+      coverageType: :instruction
```

Comment:
I personally would prefer to use `:line` as the default, what do you think?

Reply:
I don't see any issue with that and it makes sense 👍